content/channels/index.textile

```diff
@@ -999,6 +999,148 @@ channelMessageSubscription.cancel();
 <p>Calling "@subscribe()@":/api/realtime-sdk/channels#subscribe for a channel "implicitly attaches":#subscribe the client to the channel as well. It is important to note that if you call @subscribe()@ followed by @unsubscribe()@, the client remains attached to the channel.</p>
 </aside>
 
+h3(#subscription-filters). Subscription filters
+
+Subscription filters enable you to subscribe to a channel and only receive messages that satisfy a filter expression.
+
+Messages are immediately streamed to clients as soon as they "attach":#attach if they have subscribe "capabilities":/auth/capabilities for that channel. Subscription filters apply server-side filtering to messages, meaning that a client will only ever receive be sent the messages that they subscribe to.
+
+Subscription filters are currently in preview status.
+
+<aside data-type="note">
+<p>Be aware that subscription filters have an additional channel and message cost:</p>
```

Comment:
I think we can remove this. This is not currently true in the preview version. Originally, we imagined we would need to do this for the GA/non-preview release, but in recent discussions we have decided not to bill customers for suppressed messages or the additional channel, and instead Ably will swallow the cost of the additional compute.